GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -83,22 +83,24 @@ extension Glia {
         features: Features = .all,
         sceneProvider: SceneProvider? = nil
     ) throws {
-        try configure(
-            with: configuration,
-            queueId: queueID,
-            visitorContext: visitorContext
```

Comment:
you have dropped `visitorContext` from the call which is totally correct, however, in the interface the visitorContext still exists. In my opinion, it can make some confusion for an integrator, perhaps we should deprecate this function and throw away `visitorContext` + we can add print, 
```swift
if visitorContext != nil { print('visitorContext has been deprecated, please pass correct visitor context during creation SDk configuration.') }
```
what do you think, does it make sense?
@gersonnoboa  @igorkravchenko @rasmustautsglia

Reply:
Makes sense to me

Reply:
Yes, makes sense to me to deprecate this method and add a new one

Reply:
Deprecated current `start` method and add new one without `visitorContext` parameter 80a88f831ce57f7cfddba7cf3733161c14af1f59